--- origsrc/thunarx-python/docs/Makefile.am	2011-01-19 02:29:14.000000000 -0600
+++ src/thunarx-python/docs/Makefile.am	2011-01-19 03:03:26.476278500 -0600
@@ -37,6 +37,7 @@ HTMLFILES =						\
 	html/class-thunarx-python-preferences-provider.html  \
 	html/class-thunarx-python-file-info.html  \
 	html/thunarx-python.devhelp
+HTML_DATA = ${HTMLFILES}
 
 CSS_FILES = $(PYGOBJECT_PYGDOCS)/style.css
 CSSdir = $(HTMLdir)
@@ -82,6 +83,9 @@ build_stamp: $(REFERENCE_DEPS) reference
 
 pdf: $(REFERENCE_DEPS) reference/builddate.xml
 	xsltproc --nonet --xinclude -o thunarx-python-ref.fo \
-		$(PDF_STYLE) reference/thunarx-python-ref.xml
+		 --path $(BUILDDIR)/reference:$(srcdir)/reference \
+		 --stringparam gtkdoc.bookname "thunarx-python" \
+		 --stringparam gtkdoc.version ${REF_VERSION} \
+		$(PDF_STYLE) $(srcdir)/reference/thunarx-python-ref.xml
 	pdfxmltex thunarx-python-ref.fo >output </dev/null
 endif
