--- origsrc/thunarx-python-0.3.0/src/Makefile.am	2011-03-25 13:22:04.000000000 -0500
+++ src/thunarx-python-0.3.0/src/Makefile.am	2011-06-02 17:02:48.097802300 -0500
@@ -29,16 +29,23 @@ thunarx_python_la_LDFLAGS =						\
 	-avoid-version							\
 	-export-dynamic							\
 	-module								\
+	-no-undefined							\
 	$(THUNARX_LDFLAGS)						\
 	$(PYTHON_LDFLAGS)						\
 	$(PYGTK_LDFLAGS)						
 
+thunarx_python_la_LIBADD =						\
+	$(THUNARX_LIBS)							\
+	$(PYTHON_LIBS)							\
+	$(PYGTK_LIBS)
+
 pyexec_LTLIBRARIES = thunarx.la
 pyexecdir = $(THUNARX_LIBDIR)/thunarx-python
 
 thunarx_la_LDFLAGS =							\
 	-module								\
 	-avoid-version							\
+	-no-undefined							\
 	-export-symbols-regex						\
 	initthunarx							\
 	$(THUNARX_LDFLAGS)						\
